docs/Lua API.rst

```diff
@@ -4415,12 +4415,29 @@ General script API
   command-line. The ``name`` argument should be the name of the script without
   its extension, as would be used on the command line.
 
-  Note that this function lets Lua errors propagate to the caller.
+  Example::
+     dfhack.run_script("armoks-blessing", "all")
```

Comment:
For example, a change like this should fix the docs build issues:
```suggestion
  Example::
  
     dfhack.run_script("armoks-blessing", "all")
     
```

Reply:
Alright, sorry for the delay, a lot to read through. Your earlier comments are completely correct. Reviewing some of the things that I wrote, they don't make a ton of sense. I have removed some of my additions, fixed the spelling issues, and updated the example to be more accurate.